platform/src/ActivityManager.js

```diff
@@ -338,7 +338,12 @@ class ActivityManager {
 
                 // Resolve the panel definition reference  
                 if ( typeof apanel.ref == "string" ){
-                    apanel.ref = this.accessPanelDef(apanel.ref);
+                    
+                    const panelDef = this.accessPanelDef(apanel.ref);
+
+                    if (panelDef != null){
+                        apanel.ref = panelDef;
+                    }
```

Comment:
Should you not report an error in this case?

Reply:
This is reported by checkPanelRefs() from the ActivityValidator, I'm trying to keep all the validation checks together to make reuse easier in the vscode extension.